README.md

```diff
@@ -81,8 +81,23 @@
 
 ## About The Project
 
+![client-flow][client-flow]
+
 [![Product Name Screen Shot][product-screenshot]](https://www.flock.io)
 
+This new FLock client interface boasts a state-of-the-art graphical user interface (GUI), offering a seamless and user-friendly platform for our esteemed users. The client, elegantly wrapped in ```Electron```, enables users to effortlessly engage with FLock's diverse features and functionalities through a visually appealing and intuitive interface. These features include:
+
+- Communicate with a smart contract on a blockchain
+- Interact with IPFS for uploading and downloading model parameters
+- Automatically stake of tokens based on the current threshold
+- Participate in a federated learning system with a clear voting and proposing mechanism
+- Flexible with any model that implements the required interface
+- Handle training and evaluation of the models
+
+You may download the desktop client of the lastest version [here](https://github.com/FLock-io/client-interface/releases/tag/v0.0.3).
+
+For more detailed technical explanation of the FLock client, see [here] (https://docs.flock.io/core-technologies/flock-client/client-deepdown).
```

Comment:
Space between [here] and (LINK) is not rendering the link properly.

Reply:
fixed thanks!!